code/modules/clothing/suits/armor.dm

```diff
@@ -247,10 +239,10 @@
 	item_state = "armor"
 	blood_overlay_type = "armor"
 	armor = list(
-		melee = 25,
+		melee = 40,
 		bullet = 50,
-		energy = 0,
-		bomb = 0,
+		energy = 30,
+		bomb = 30,
```

Comment:
Platecarrier shouldn't rock better bomb resistance than armors dedicated against it, such as the flak.

Reply:
Honestly idk what the difference between a plate carrier and flak vest is, pretty sure I just made them identical stat wise cause they were pretty similar ingame.

Reply:
> Honestly idk what the difference between a plate carrier and flak vest is, pretty sure I just made them identical stat wise cause they were pretty similar ingame.

Flak vest costs more in the serb vendor for more protection. Dont make it the fucken same

Reply:
Flak vest was actually underpowered, it had much less bullet protection in exchange for more bomb protection

Reply:
Ok so what do you recommend for plate carrier to put it between a flak vest and a regular vest.

Reply:
> Ok so what do you recommend for plate carrier to put it between a flak vest and a regular vest.

Platecarrier ideally should only be better at bullet than flak.

Reply:
But I can't get any better at bullet without it just being BP.

Reply:
Resolved, it has far more rounded out stats than the flak now with less bomb protection.

Also just hear me out, I thought a flak vest was meant to protect you from shrapnel, which is covered by bullet resistance. Idc either way though.